src/test/java/ExecutorServiceTest.java

```diff
@@ -0,0 +1,28 @@
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.LongStream;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import project.SumExecutorService;
+
+public class ExecutorServiceTest {
+    private static final long SUM_OF_NUMBERS = 500000500000L;
```

Comment:
```suggestion
    private static final long SUM_OF_NUMBERS = 500_000_500_000L;
```
там більш читабельно